docs/website/blog/2022-05-02-odo-v3-alpha1.md

```diff
@@ -0,0 +1,319 @@
+
+Watching for changes in the current directory /home/phmartin/Documents/tests/devto-deploy
+Press Ctrl+c to exit `odo dev` and delete resources from the cluster
+```
+
+To be able to debug the application, you will need to run the `odo dev --debug` command instead.
+
+When you have finished the development session, you just need to hit Ctrl-c to stop the `odo dev` command. The command won't terminate immediately, as it will delete the resources it has deployed on the cluster before to exit.
+
+## The deployment phase
+
+When you are satisfied with your program, you may want to deploy it. The first step would be to build the container image using a `Dockerfile`, instead of using a generic image as during the development phase. The second step would be to deploy personalized resources, instead of the Deployment used during the development phase.
+
+At the time of this blog post, no Devfile within the default Devfile registry contains instructions for the deployment phase. By using the Devfile provided as example above, the command `odo deploy` will deploy a personalized Deployment and a Service into the cluster, after building the container image using the `Dockerfile` present in the directory.
+
+```
+$ odo deploy
+  __
+ /  \__     Deploying the application using my-nodejs-app Devfile
+ \__/  \    Namespace: prj2
+ /  \__/    odo version: v3.0.0-alpha1
+ \__/
+
+↪ Building & Pushing Container: quay.io/phmartin/myimage
+ •  Building image locally  ...
+STEP 1/7: FROM docker.io/library/node:17
+STEP 2/7: WORKDIR /usr/src/app
+[...]
+STEP 7/7: CMD [ "node", "server.js" ]
+COMMIT quay.io/phmartin/myimage
+ ✓  Building image locally [6s]
+ •  Pushing image to container registry  ...
+[...]
+Writing manifest to image destination
+Storing signatures
+ ✓  Pushing image to container registry [8s]
+
+↪ Deploying Kubernetes Component: my-node
+ ✓  Searching resource in cluster 
+ ✓  Creating kind Deployment [50ms]
+
+↪ Deploying Kubernetes Component: svc
+ ✓  Searching resource in cluster 
+ ✓  Creating kind Service [57ms]
+
+Your Devfile has been successfully deployed
+```
+
+At any moment, you can check if a component has been deployed by using the `odo list` command.
+
+```
+$ odo list
+ ✓  Listing components from namespace 'prj2' [61ms]
+ NAME             PROJECT TYPE  RUNNING IN  MANAGED 
+ * my-nodejs-app  nodejs        Deploy      odo     
+```
+
+When you are done with this application or if you want to undeploy it to work on development mode again, you can use the `odo delete component` to undeploy the component from the cluster.
+
+```
+$ odo delete component
+Searching resources to delete, please wait...
+This will delete "my-nodejs-app" from the namespace "prj2".
+ •  The component contains the following resources that will get deleted:
+	- Deployment: my-node
+	- Service: svc
+? Are you sure you want to delete "my-nodejs-app" and all its resources? Yes
+The component "my-nodejs-app" is successfully deleted from namespace "prj2"
+```
```

Comment:
```suggestion

## End-to-end use case
<iframe width="740" height="500" src="https://www.youtube.com/embed/fw2_cVj3Mv4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

@feloy I couldn't push to your repository, so I am making a suggestion here. Please verify the changes after you implement the suggestion. Also, if you can suggest a better words than `End-to-end use case`, even better!